### PR TITLE
DRK Darkside tracking and bugfixes

### PR DESCRIPTION
**Pull request by vond82**
- DRK Darkside tracking since it's been requested a lot
- Moving PLD buffbar no longer moves all PLD job elements
- Primary Resource label can now be properly jobcolored

### CODE REVIEW

**Comment by vond82** [comment]
Converted to draft cause i need to fix some positioning for DRK I just realized, just no time

File: DelvUI/Interface/Jobs/DarkKnightHud.cs [hunk 1]
```diff
@@ -43,6 +43,12 @@ protected override (List<Vector2>, List<Vector2>) ChildrenPositionsAndSizes()
                 sizes.Add(Config.BloodGaugeSize);
             }
 
+            if (Config.ShowDarkside)
+            {
+                positions.Add(Config.Position + Config.DarksidePosition);
+                sizes.Add(Config.DarksideSize);
+            }
+
             if (Config.ShowBuffBar)
```
Comment (Tischel) [inline DelvUI/Interface/Jobs/DarkKnightHud.cs:52]: Probably shouldnt add this bar for draggable area calculations since its not always showing.

Comment (vond82) [reply DelvUI/Interface/Jobs/DarkKnightHud.cs:52]: I planned on making it optional wether the bar always shows or "only show when active" much like Zeff did with bard dots. My plan was actually to get around to making that an option for most elements in the jobpacks that are triggered and have a timer

**Comment by vond82** [comment]
[Moving PLD FoF&Requiescat bar moves every PLD element](https://trello.com/c/HQSIaPUg/186-moving-pld-fofrequiescat-bar-moves-every-pld-element)

**Comment by vond82** [comment]
[Job Color for Labels/Primary Resource](https://trello.com/c/PHMoUigc/177-job-color-for-labels-primary-resource)

**Comment by vond82** [comment]
[Add Darkside track to DRK pack](https://trello.com/c/NiemyWfL/133-add-darkside-track-to-drk-pack)

**Review by jdsmith2816 — APPROVED** [review]

**Pull request merged**